setup.py

```diff
@@ -41,7 +41,7 @@
     install_requires=[
         "ugrc-palletjack>=5.0,<5.3",
         "ugrc-supervisor>=3.1.3",
-        "h3==3.*",
+        "h3>=3,<5",
```

Comment:
I'm not an expert on how python resolved dependencies, but doesn't this mean that this project should be compatible with h3 v4 & v3? If so, are these changes that you made compatible with v3?
```suggestion
        "h3==4.*",
```

Reply:
Good catch. The new code is 4.x only,

Reply:
Good catch, the code changes are 4.x only.